src/include/test/unit_vamana_index.cc

```diff
@@ -656,13 +703,16 @@ TEST_CASE("vamana: diskann fbin", "[vamana]") {
     // the start point should be included in the top_k list.  It probably
     // should not be, so comment out for now.
     // CHECK(top_k[0] == start);
-    CHECK(std::find(begin(top_k), end(top_k), start) != end(top_k));
-    CHECK(std::find(begin(V), end(V), start) != end(V));
+    // CHECK(std::find(begin(top_k), end(top_k), start) != end(top_k));
+    // CHECK(std::find(begin(V), end(V), start) != end(V));
```

Comment:
@lums658 Can we add this back? Why remove these checks?

Reply:
They were commented out pending determination of whether or not the start should be included (to be consistent with DiskANN).  Uncommented.

Reply:
Great, thanks